.eslintrc.json

```diff
@@ -3,7 +3,18 @@
   "env": {
     "browser": true
   },
+  "parser": "babel-eslint",
   "rules": {
+    "arrow-parens":0,
+    "arrow-body-style":0,
+    "no-named-as-default":0,
+    "jsx-a11y/anchor-is-valid":["enabled", "never"],
+    "react/button-has-type":["enabled", "never"],
+    "jsx-a11y/no-static-element-interactions":["enabled", "never"],
+    "jsx-a11y/click-events-have-key-events":["enabled", "never"],
+    "import/no-named-as-default":["enabled", "never"],
+    "react/no-unescaped-entities": ["enabled", "never"],
+    "react/destructuring-assignment": ["enabled", "never"],
```

Comment:
why did you disable this rule?

Reply:
They were not very necessary.

Reply:
They don't have to be "very" necessary. Destructuring is a javascript best practice

Reply:
Alright. Let me work on it.